src/handlers/patient_handler.go

```diff
@@ -0,0 +1,109 @@
+package handlers
+
+import (
+	"clinic-api/src/models"
+	"clinic-api/src/utils"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/google/uuid"
+	"github.com/labstack/echo/v4"
+)
+
+func CreatePatientHandler(c echo.Context) error {
+	status := http.StatusCreated
+	patientRequest := models.Patient{}
+	if err := c.Bind(&patientRequest); err != nil {
```

Comment:
If this directly bind to the model, it can be error if `dob` got from `string` type request, as created in html form. Prefer create a new struct for request then map into model.